sphinx_gallery/gen_gallery.py

```diff
@@ -364,6 +365,7 @@ def _fill_gallery_conf_defaults(sphinx_gallery_conf, app=None, check_keys=True):
     for key in (
         "promote_jupyter_magic",
         "run_stale_examples",
+        "embed_code_links",
     ):
         gallery_conf[key] = _bool_eval(gallery_conf[key])
```

Comment:
Reminder to myself, I should add a test checking `_bool_eval` works as expected.